crates/shielder_bindings/README.md

```diff
@@ -0,0 +1,45 @@
+# Shielder Bindings
+
+Platform-native bindings for the Shielder cryptographic operations, supporting both WebAssembly (WASM) and mobile platforms through a unified Rust codebase.
+
+## Overview
+
+The shielder_bindings crate provides a no_std compatible interface to the Shielder cryptographic operations. It supports multiple compilation targets including WebAssembly for browser environments and platform-native bindings (via UniFFI) for mobile platforms.
+
+## Building
+
+### Prerequisites
+
+- Rust toolchain (see rust-toolchain.toml)
+- wasm-pack (for WebAssembly builds)
+- Rust target: aarch64-apple-ios-sim (for iOS simulator builds)
+
```

Comment:
something more, because when I do `make ios` I get `error: failed to get iphonesimulator SDK path: No such file or directory (os error 2)`